### DISCUSSION TITLE
Map Layout/Interface Toolbar Buttons Should have Tooltips

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#9323] [Feature Requests]
While the button icons are relatively clear, they should still have tooltips. But apparently, this is made difficult by the mapping library. CC: @Oreilles 

Ref: https://github.com/directus/directus/issues/7895

![Screen Shot 2021-11-01 at 4 40 27 PM](https://user-images.githubusercontent.com/522079/139739012-214ddf5c-d81f-48be-a156-285d40b9af14.png)

**Comment by licitdev** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
